draft-venhoek-nts-pool.md

```diff
@@ -174,6 +174,16 @@ A client MAY send multiple of these records if desired. The data in the record S
 
 MUST NOT be sent by a server. Server MAY at its discretion ignore the request from the client and still provide the given server in an NTPv4 Server Negotiation record.
 
+## Authentication {#authentication}
```

Comment:
Should we add anything about the length of the ASCII string (i.e. security considerations, what is an acceptable length)? Are there any min or max lengths implementations should/must accept?